src/nurgling/actions/EquipFromInventory.java

```diff
@@ -0,0 +1,77 @@
+package nurgling.actions;
+
+import haven.Coord;
+import haven.WItem;
+import nurgling.NGItem;
+import nurgling.NGameUI;
+import nurgling.NInventory;
+import nurgling.NUtils;
+import nurgling.tasks.WaitItemInEquip;
+import nurgling.tools.NAlias;
+import nurgling.widgets.NEquipory;
+
+public class EquipFromInventory implements Action {
```

Comment:
I think this should be EquipFromContainer and all of the code for equipping from inventory removed from here, but im not sure.